indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
+       'editable': editable ? 1 : 0
+    };
+    var balloonId = '#qtip-' + entryId.toString();
+
+    $(timetableBlock).qbubble({
+       id: entryId.toString(),
+       content: {
+           text: function(evt, api) {
+               $.ajax({
+                   url: build_url(Indico.Urls.Timetable.entries.balloon, params)
+               })
+               .then(function(content) {
+                   // Set the tooltip content upon successful retrieval
+                   api.set('content.text', content.html);
+                   var $content = api.elements.content;
+                   if (editable) {
+                       $duration = $content.find('.js-edit-time');
+                       $duration.ajaxqbubble({
+                           url: build_url(Indico.Urls.Timetable.entries.editTime, params),
+                           qBubbleOptions: {
+                               style: {
+                                   classes: 'balloon-time-qtip'
+                               },
+                               position: {
+                                   at: 'top center',
+                                   my: 'bottom center',
+                                   target: $(timetableBlock)
+                               }
+                           },
+                           onClose: function(data) {
+                               if (data.entries) {
+                                   self.managementActions._addEntries(data.entries);
+                               }
+                           }
+                       });
+
+
+                       $content.find('.js-delete').on('click', function() {
+                           self.managementActions.deleteEntry(self.eventData);
+                       });
+
+                       $content.find('.js-edit').on('click', function() {
+                           var extraParams = $(this).data('extra-params');
+                           jQuery.extend(params, extraParams);
+                           ajaxDialog({
+                               trigger: this,
+                               url: build_url(Indico.Urls.Timetable.entries.edit, params),
+                               title: $(this).data('title'),
+                               onClose: function(data) {
+                                   if (data && data.entries) {
+                                       self.managementActions._addEntries(data.entries);
+                                   }
+                               }
+                           });
+                       });
+                   }
+                   $content.find('.close-balloon').each(function() {
+                       $(this).on('click', function() {
+                           $(timetableBlock).qtip('hide');
+                       });
+                   });
+                   // Change the target of the qTip position in order to open it at the mouse position
+                   api.options.position.target = 'mouse';
```

Comment:
Hm, based on the docs, it's none of our proposals, it looks like it's this:
```javascript
api.set('position.targer', 'mouse')
```
Not much of a difference, but I'll change it.